server/bundler/loader.js

```diff
@@ -102,7 +146,11 @@ function splitTemplate( path, section ) {
 		'		context.store.dispatch( { type: "SECTION_SET", isLoading: false } );',
 		'		controller.setSection( ' + JSON.stringify( section ) + ' )( context );',
 		'		if ( ! _loadedSections[ ' + JSON.stringify( section.module ) + ' ] ) {',
-		'			require( ' + JSON.stringify( section.module ) + ' )( controller.clientRouter );',
+		'			require( ' + JSON.stringify( section.module ) + ' )( ',
```

Comment:
Shouldn't we load the JS from a callback of the JS, rather than the other way round?